chroma-manager/requirements.dev

```diff
@@ -18,4 +18,4 @@ pyflakes
 pygraphviz
 sphinx==1.1.3
 werkzeug
-iml-common
+iml-common>=1.0.5
```

Comment:
Just curious why we need to pin this?

Just because, I think generally speaking, we'd like to get away from pinning since we are so bad at updating the pins to something recent and we then get into a place where we are unknowingly using old, vulnerable versions of modules.